README.md

```diff
@@ -36,6 +49,33 @@ Return to your original locations.  In this step all groups will act like Design
 
 ***Q1.3 Briefly describe your solutions (use a list) and any strengths/weaknesses you see.***
 
+partner with school in question (access student information, student gives permission)
+
+weather information - where school is located
+gps information of school
+
+body type/activity level of student (survey at installation of program) - maybe connect to water bottle? tick marks on how much water to drink (water bottle product is linked to application)
+
+Services:
+* Traffic/public transit - will I get water in time (train stations have water fountains?)
+* Maps - gps location for water fountains
+* Weather - as stated above
+* Health and Fitness: maybe connect to fitbit, measure activity level throughout today
```

Comment:
Getting notifications from a watch about your vitals and how much water you need would definitely be feasible. Such a device might help students get in the habit of drinking more water.